.github/workflows/publish.yml

```diff
@@ -320,7 +320,11 @@ jobs:
         uses: actions/checkout@v3
         with:
           ref: ${{ inputs.git_ref }}
-          submodules: "recursive"
+        # Workaround for older OS images
+        # https://github.com/actions/checkout/issues/758
+      - name: Checkout submodules
+        run: |
+          git submodule update --init --recursive
```

Comment:
nit

```suggestion
        run: git submodule update --init --recursive
```

Reply:
Eh, I prefer using the pipe for intentionality given the general trickiness of YAML - but happy to change it if that's a dissenting opinion.